.travis.yml

```diff
@@ -69,6 +69,11 @@ matrix:
       env:
         - "JRUBY_OPTS='--dev -J-Xmx1024M'"
         - "GEM='am'"
+    - rvm: jruby-9.1.5.0
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='--dev -J-Xmx1024M'"
```

Comment:
Per [your comment](https://github.com/rails/rails/pull/27074#discussion_r88349688) @matthewd, the only place we'd be able to "globally" specify this information is at the complete global level, so this env var would be set for all builds. If you're okay with that then I can make a patch to do that 😬